frontend/app/src/comps/ManagedInterestRateField/ManagedInterestRateField.tsx

```diff
@@ -0,0 +1,1157 @@
+                            )
+                            .with(PublicStatus.ChangePlanned, () => {
+                              const rawRate = selectedManagedDelegate.status.targetInterestRate;
+                              const ir = fmtnum(rawRate / 1e18, "pct2");
+                              
+                              return (
+                                <>
+                                  Interest rate adjustment to <b>{ir}%</b> predicted{' '}
+                                  {selectedManagedDelegate.status.daysToAdjustment === 0
+                                    ? 'today'
+                                    : `in ${selectedManagedDelegate.status.daysToAdjustment} days`}
+                                  .
+                                </>
+                              );
+                            })
+                            .with(PublicStatus.Deprecated, PublicStatus.Inactive, () => {
+                              return <>This strategy is not active.</>;
+                            })
+                            .exhaustive()
+                        }
+                      </div>
+                    </div>
+                  </div>
+                </a.div>
+              ) : null}
+            </>
+          }
+          {...fieldValue.inputFieldProps}
+          onFocus={() => {
+            setIsInputFocused(true);
+            fieldValue.inputFieldProps.onFocus?.();
+          }}
+          onBlur={() => {
+            setIsInputFocused(false);
+            fieldValue.inputFieldProps.onBlur?.();
+          }}
+          value={
+            // no delegate selected yet
+            (mode !== "manual" && delegate === null)
+              ? ""
+              : fieldValue.value
+          }
+          valueUnfocused={
+            (mode !== "manual" && delegate === null)
+              ? null
+              : interestRate
+                ? (
+                  <span
+                    style={{
+                      display: "flex",
+                      alignItems: "center",
+                      gap: 10,
+                    }}
+                  >
+                    {delegate !== null && <MiniChart size="medium" />}
+                    <span style={{ fontVariantNumeric: "tabular-nums" }}>
+                      {fmtnum(interestRate, "pct1z")}
+                    </span>
+                    <span style={{ color: "#878AA4", fontSize: 24 }}>
+                      % per year
+                    </span>
+                  </span>
+                )
+                : null
+          }
+        />
+
+        <DelegateModal
+          branchId={branchId}
+          onClose={() => setDelegatePicker(null)}
+          onSelectDelegate={handleDelegateSelect}
+          visible={delegatePicker === "delegate"}
+        />
+      </>
+    );
+  },
+  (prev, next) => jsonStringifyWithDnum(prev) === jsonStringifyWithDnum(next),
+);
+
+function ManualInterestRateSlider({
```

Comment:
Small color issue of the slider line. Not visible on the right side of value.

How it should look:
<img width="558" height="179" alt="Screenshot 2025-07-22 at 1 31 46 PM" src="https://github.com/user-attachments/assets/ad98a787-d385-4ab7-95ed-20d844f5aa3d" />

How it looks here:
<img width="557" height="178" alt="Screenshot 2025-07-22 at 1 31 24 PM" src="https://github.com/user-attachments/assets/5c594fa3-5589-4037-a31e-da393dc44749" />